xml/art_migration.xml

```diff
@@ -840,48 +850,51 @@ mig-cp1-neut-m1-mgmt</screen>
 <prompt>(seed)> </prompt>sudo systemctl stop cobblerd
 <prompt>(seed)> </prompt>sudo systemctl disable cobblerd
 <prompt>(seed)> </prompt>sudo sed -i /sles12sp3.iso/d /etc/fstab
-<prompt>(seed)> </prompt>sudo umount /media/cdrom
-<prompt>(seed)> </prompt>sudo sed -i /sles12sp3.iso/d /etc/fstab</screen>
+<prompt>(seed)> </prompt>sudo umount /media/cdrom</screen>
+     </listitem>
+     <listitem>
+      <para>
+       If you used the cobbler mechanism from your seed node earlier to re-image
+       the node which will now be serving as your new deployer, its zypper
+       repos should be changed to use local sources instead of retrieving content
+       from the seed node via the network:
+      </para>
+      <screen><prompt>(new-deployer)> </prompt>sudo zypper rr SLES-OS
+<prompt>(new-deployer)> </prompt>sudo zypper ar file:/media/cdrom SLES12-SP3-12.3-0</screen>
+      <para>
+       The old repos on other &slsa; systems will automatically be reset to the
+       new deployer later when the <filename>osconfig-run.yml</filename> playbook is run.
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       Copy the user's home directory and certificates from the seed to the new deployer:
+      </para>
+<screen><prompt>(seed)> </prompt>rsync -avP ~/ <replaceable>NEW_DEPLOYER_IP</replaceable>:/home/stack
+<prompt>(seed)> </prompt>scp -r /tmp/hel* <replaceable>NEW_DEPLOYER_IP</replaceable>:/tmp</screen>
      </listitem>
      <listitem>
       <para>
-       Copy the user's home directory from the seed to the new deployer:
+       Unmount any ISOs which are no longer necessary on the seed node:
       </para>
-<screen><prompt>(seed)> </prompt>rsync -avP ~/ <replaceable>NEW_DEPLOYER_IP</replaceable>:/home/stack</screen>
+<screen><prompt>(seed)> </prompt>sudo umount /home/stack/hos5.scp</screen>
      </listitem>
      <listitem>
       <para>
        Connect to the new deployer and initialize it for use:
       </para>
 <screen><prompt>(new-deployer)> </prompt>cd ~
+<prompt>(new-deployer)> </prompt>rm -rf .ansible-fact-cache
 <prompt>(new-deployer)> </prompt>sudo mkdir -p /media/cdrom
-<prompt>(new-deployer)> </prompt>echo /home/stack/sles12sp3.iso /media/cdrom iso9660 loop 0 0 \
-  | sudo tee -a /etc/fstab
-<prompt>(new-deployer)> </prompt>sudo mount -a
+<prompt>(new-deployer)> </prompt>sudo mount /home/stack/sles12sp3.iso /media/cdrom
+<prompt>(new-deployer)> </prompt>sudo mount HPE-HELION-OPENSTACK-5-TO-8-MIGRATION-SCP-x86_64-991.iso hos-5.scp
 <prompt>(new deployer)> </prompt>tar -xf hos-5.scp/hos/hos-5.1.0-*.tar
 <prompt>(new deployer)> </prompt>cd ~/hos-5.1.0
 <prompt>(new deployer)> </prompt>./hos-init.bash
```

Comment:
this is where we see the error it the new deployer is not standalone but hosted on a controller. we need to verify if this occurs all of the time.

TASK: [KEY-API | status | Register keystone status] ***************************
fatal: [localhost] => One or more undefined variables: 'host' is undefined

FATAL: all hosts have already failed -- aborting

a cp/readty resolves it, maybe a refresh facts as well

Reply:
This code I added should resolve the issue:
rm -rf .ansible-fact-cache